components/kyma-environment-broker/internal/broker/instance_create.go

```diff
@@ -103,12 +118,17 @@ func (b *ProvisionEndpoint) Provision(ctx context.Context, instanceID string, de
 		return b.handleExistingOperation(existingOperation, provisioningParameters, logger)
 	}
 
+	// create SKR shoot name
+	shootName := gardener.CreateShootName()
+	dashboardURL := fmt.Sprintf("https://console.%s.%s.%s", shootName, b.shootProject, strings.Trim(b.shootDomain, "."))
+
 	// create and save new operation
 	operation, err := internal.NewProvisioningOperationWithID(operationID, instanceID, provisioningParameters)
 	if err != nil {
 		logger.Errorf("cannot create new operation: %s", err)
 		return domain.ProvisionedServiceSpec{}, errors.New("cannot create new operation")
 	}
+	operation.ShootName = shootName
```

Comment:
in the XSUAA step I need also full domain: "{shootName}.{shootProject}.{shootDomain}"